observability/prometheus/http.go

```diff
@@ -64,13 +64,6 @@ func newHttpHandler(registry prometheus.Registerer, prefix string, URLPrefix str
 		[]string{},
 	)
 
-	registry.MustRegister(
-		handler.inFlightGauge,
-		handler.counter,
-		handler.durationSeconds,
-		handler.responseSizeBytes,
-	)
-
```

Comment:
Do you know why these are getting deleted?

Reply:
I was reading up on the .MustRegister method and found that promauto would take care of registering given a registry. As this was not the origin of the error I dropped promauto again. But forgot about reverting this. 

Will fix